source/overview/auth.rst

```diff
@@ -35,11 +35,12 @@ AD/LDAP is the most popular corporate directory integration option for deploying
 - AD/LDAP user filters to define which users get access to Mattermost in the form of a query
 - Ability to use a low-privileged AD/LDAP account to run queries over a secure TLS or STARTTLS connection
 - Attribute mapping to place First Name, Last Name, Nickname and other attributes from AD/LDAP into Mattermost 
+- Customization of login screen to specify whether users are logging in with email, username or other attribute
 - Sychronization with AD/LDAP to disable, enable and update Mattermost users based on AD/LDAP 
 
-For very large AD/LDAP instances you can also configure max page size to divide a Mattermost AD/LDAP query into several pieces to not over-tax the authentication server when synchronizing.
+New user accounts are created when new users login with their AD/LDAP credentials. If you're using email or username and password authentication `users can switch to AD/LDAP manually <https://docs.mattermost.com/deployment/sso-ldap.html#configure-ad-ldap-using-the-system-console-user-interface>`_, and the `conversion to AD/LDAP can also be done using the command line interface <https://docs.mattermost.com/administration/command-line-tools.html?highlight=cli#platform-user-migrate-auth>`_ by an IT admin. 
 
-For private network deployments in development teams using GitLab for source control, `GitLab single sign-on <https://docs.mattermost.com/deployment/sso-gitlab.html>`_ is also available. 
+For very large AD/LDAP instances you can also configure max page size to divide a Mattermost AD/LDAP query into several pieces to not over-tax the authentication server when synchronizing.
```

Comment:
I think max page size is more for if the LDAP server restricts the number of users that can be requested at one time